chess/src/components/Tile.tsx

```diff
@@ -0,0 +1,23 @@
+import TileCSS from'./Tile.module.css';
+
+interface Props {
+  number: number,
+  image?: string,
+}
+
+export default function Tile({number, image}: Props) {
```

Comment:
there is a good resource on [typescipt in react](https://github.com/typescript-cheatsheets/react)

I would use `React.PureComponent<Props, State> ` here